README.md

```diff
@@ -172,6 +172,18 @@ This event was mislabeled \#46 on Meetup.
 
 2012-12-11 | 18:30 | [Meetup Event](https://www.meetup.com/Berlin-Hack-and-Tell/events/94195282/) | [Photos](https://www.meetup.com/Berlin-Hack-and-Tell/photos/12140712/) | [@c-base](https://www.c-base.org)
 
+* Bomberman multiplayer 3d by Stefan
+* [sq.rotogame.com (rotating puzzle game in JS/node/socket io)](https://github.com/feklee/rotogamesq) by [Felix](http://twitter.com/feklee)
+* [NoFlo (JavaScript implementation of Flow-Based Programming)](https://github.com/noflo/noflo) by [Henri](https://github.com/bergie)
```

Comment:
I created a pull request for that: #58

@bergie If you see more things to improve, just create more pull requests.